circle.yml

```diff
@@ -3,8 +3,14 @@ machine:
   php:
     # https://circleci.com/docs/environment#php
     version: 5.5.11
+  pre:
+    # @todo look for a better way to install phantomjs.
+    - sudo curl --output /usr/local/bin/phantomjs https://s3.amazonaws.com/circle-downloads/phantomjs-2.1.1
```

Comment:
The common pattern for doing things like this on circle seems to be:

1. download the artifact into a `~/deps`, add that dir to the circle.yml cache_directories so that it is available on future builds. If it's already there (and matches your expected version), don't re-download it.
2. install from `~/deps` to somewhere in the $PATH. You could put it into /usr/local/bin or put it in `~/bin` and ensure ~/bin is in the PATH (you can set PATH in circle.yml)

We have several examples throughout our codebase of this. Here is one I recently used in a non-pantheon project:

https://github.com/joemiller/creds/blob/master/test/circle_deps.sh#L9-L18
https://github.com/joemiller/creds/blob/master/circle.yml#L3-L7